gamestonk_terminal/forex/fx_view.py

```diff
@@ -0,0 +1,544 @@
+        with open(".candles.csv", "w") as out:
+            out.write("Datetime, Open, High, Low, Close, Volume\n")
+        for line in lines:
+            with open(".candles.csv", "a") as output:
+                output.write(line[:10] + " " + line[11:19] + line[30:])
+
+
+def calendar(instrument, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="calendar",
+        description="Show Calendar Data",
+    )
+    parser.add_argument(
+        "-d",
+        "--days",
+        dest="days",
+        action="store",
+        type=int,
+        default=7,
+        required=False,
+    )
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+
+    parameters = {"instrument": instrument, "period": str(ns_parser.days * 86400 * -1)}
+    try:
+        request = labs.Calendar(params=parameters)
+        response = client.request(request)
+        for i in range(len(response)):
+            if "title" in response[i]:
+                title = response[i]["title"]
+                print(f"Title: {title}")
+            if "timestamp" in response[i]:
+                timestamp = response[i]["timestamp"]
+                time = datetime.fromtimestamp(timestamp)
+                print(f"Time: {time}")
+            if "impact" in response[i]:
+                impact = response[i]["impact"]
+                print(f"Impact: {impact}")
+            if "forecast" in response[i]:
+                forecast = response[i]["forecast"]
+                unit = response[i]["unit"]
+                if unit != "Index":
+                    print(f"Forecast: {forecast}{unit}")
+                else:
+                    print(f"Forecast: {forecast}")
+            if "market" in response[i]:
+                market = response[i]["market"]
+                unit = response[i]["unit"]
+                if unit != "Index":
+                    print(f"Market Forecast: {market}{unit}")
+                else:
+                    print(f"Market Forecast: {market}")
+            if "currency" in response[i]:
+                currency = response[i]["currency"]
+                print(f"Currency: {currency}")
+            if "region" in response[i]:
+                region = response[i]["region"]
+                print(f"Region: {region}")
+            if "actual" in response[i]:
+                actual = response[i]["actual"]
+                unit = response[i]["unit"]
+                if unit != "Index":
+                    print(f"Actual: {actual}{unit}")
+                else:
+                    print(f"Actual: {actual}")
+            if "previous" in response[i]:
+                previous = response[i]["previous"]
+                unit = response[i]["unit"]
+                if unit != "Index":
+                    print(f"Previous: {previous}{unit}")
+                else:
+                    print(f"Previous: {previous}")
+            print("-" * 30)
```

Comment:
print("") at the end.

I'll stop commenting this, but ideally all commands end up with a new line :)